### DISCUSSION TITLE
Updating Github Pages Deployment

### DISCUSSION THREAD

**Discussion by joej164** [getpelican/pelican#3140] [Ideas]
Github Pages Allows deploying from either the root of a repo or from the /docs folder.

I updated my `pelicanconf.py` file to contain the line `OUTPUT_PATH = "docs"` and I then pushed the whole project and the site deployed very easily.

This made the deployment process for USER pages very simple.  Any objections to me doing a PR to update the docs to include this option as well?

**Comment by justinmayer** [comment]
@joej164: Sounds good to me. Looking forward to your docs enhancement PR! 😊
